src/index.html

```diff
@@ -13,5 +13,217 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post"
+    >
+
+      <fieldset class="groups">
+        <legend>Personal information</legend>
+        
+          <label class="form-field">
+            Surname:
+             <input class="inputs" 
+                type="text" 
+                name="Surname"
+                required
+                autocomplete="off"
+                >
```

Comment:
```suggestion
             <input 
               class="inputs" 
               type="text" 
               name="Surname"
               required
               autocomplete="off"
             >
```
There should be 2 spaces indent

Reply:
- [DEMO LINK](https://denys-sadovyi.github.io/layout_html-form/)
 - [TEST REPORT LINK](https://denys-sadovyi.github.io/layout_html-form/report/html_report/)